renderer/containers/action-bar.js

```diff
@@ -17,18 +17,18 @@ export default class ActionBarContainer extends Container {
 
     this.settings = this.remote.require('./common/settings');
     this.state = {
-      cropperWidth: 0,
-      cropperHeight: 0
+      cropperWidth: '0',
+      cropperHeight: '0'
```

Comment:
Why did you go with saving them as strings and not numbers? Just curious. I haven't looked closely at the code yet. Just seems like the obvious would be to store as numbers.

Reply:
Because the validation is `debounce`d, so the user could enter a string that is not a number. Those values there are what the user sees in the input field. The `width` and `height` in the other container are the actual dimensions of the cropper. And they are synced when the debounced func runs